kpi-queries/src/main/java/com.c12e.cortex.examples.aggregate/KPIQueries.java

```
@@ -0,0 +1,288 @@
+                "                  \"windowDuration\"\n" +
+                "                ],\n" +
+                "                \"connection\": {\n" +
+                "                  \"name\": \"KPI-" + name + "\"\n" +
+                "                },\n" +
+                "                \"description\": \""+description+"\",\n" +
+                "                \"kind\": \"batch\",\n" +
+                "                \"name\": \"KPI-" + name + "\",\n" +
+                "                \"primaryKey\": \"timeOfExecution\",\n" +
+                "                \"title\": \"" + name + "\"\n" +
+                "              }";
+
+        String connectionConfig = "{\n" +
+                "                \"project\": \"" + project + "\",\n" +
+                "                \"name\": \"KPI-" + name + "\",\n" +
+                "                \"title\": \"" + name + "\",\n" +
+                "                \"connectionType\": \"gcs\",\n" +
+                "                \"contentType\": \"parquet\",\n" +
+                "                \"allowRead\": true,\n" +
+                "                \"allowWrite\": false,\n" +
+                "                \"params\": [\n" +
+                "                  {\n" +
+                "                    \"name\": \"uri\",\n" +
+                "                    \"value\": \"gs://"+profilesBucket+"/sources/"+project+"/KPI-"+name+"-delta\"\n" +
+                "                  },\n" +
+                "                  {\n" +
+                "                    \"name\": \"workloadIdentityEnabled\",\n" +
+                "                    \"value\": \"true\"\n" +
+                "                  }\n" +
+                "                ]\n" +
+                "              }";
+
+        Double value = null;
+        try {
+            value = runKPI(cortexSession, project);
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Is this try/catch meant to keep the program going if the `script` argument is not parsable? I would think we should throw an error if the script isn't valid.. (I'm assuming that a `ParseException` is thrown whilst parsing the profile-script).